config/crd/bases/datasciencepipelinesapplications.opendatahub.io_datasciencepipelinesapplications.yaml

```diff
@@ -541,7 +541,6 @@ spec:
                           secretKey:
                             type: string
                           secretName:
-                            description: 'Note: In V2 this value needs to be mlpipeline-minio-artifact'
```

Comment:
I think you might need to `make generate` to update the new description in `dspipeline_types.go` in this CRD yaml file.

Reply:
`make generate` didn't make any changes.

Reply:
hmm. Something should be updating the comment :thinking:

Reply:
`make manifests` worked.
I just pushed the changes.